iroh-p2p/src/swarm.rs

```diff
@@ -77,15 +74,32 @@ async fn build_transport(
 
         (transport, Some(relay_client))
     } else {
-        let transport = transport
+        let tcp_transport = tcp_transport
             .upgrade(core::upgrade::Version::V1Lazy)
             .authenticate(auth_config)
             .multiplex(muxer_config)
-            .timeout(connection_timeout)
             .boxed();
 
-        (transport, None)
-    }
+        (tcp_transport, None)
+    };
+
+    // TODO: reenable websocket if we still want it
+    // let websocket_transport = websocket::WsConfig::new(tcp::tokio::Transport::new(tcp_config));
```

Comment:
@elenaf9  haven't gotten websocket integrated, not sure how to

Reply:
What issues are you running into?

Reply:
no matter how I combine the ortransports the compiler is upset with me 🥲

Reply:
@elenaf9 any ideas how to set this up?

Reply:
Just tried it locally, the below works for me with no issues:
```rust
let tcp = tcp::tokio::Transport::new(tcp::Config::new());
let ws_tcp = websocket::WsConfig::new(tcp::tokio::Transport::new(tcp::Config::new()));

let tcp_relay_transport = OrTransport::new(relay_transport, tcp.or_transport(ws_tcp))
    .upgrade(upgrade::Version::V1)
    .authenticate(noise::NoiseConfig::xx(noise_keys).into_authenticated())
    .multiplex(libp2p::yamux::YamuxConfig::default());
let quic_transport = quic::tokio::Transport::new(quic::Config::new(&local_key));

let transport = OrTransport::new(quic_transport, tcp_relay_transport)
    .map(|either_output, _| match either_output {
        EitherOutput::First((peer_id, muxer)) => (peer_id, StreamMuxerBox::new(muxer)),
        EitherOutput::Second((peer_id, muxer)) => (peer_id, StreamMuxerBox::new(muxer)),
    })
    .boxed();
```

Reply:
thank you 🎉  I inserted it at the wrong position before